#################################################################
#								#
# Copyright (c) 2022 YottaDB LLC and/or its subsidiaries.	#
# All rights reserved.						#
#								#
#	This source code contains the intellectual property	#
#	of its copyright holder(s), and is made available	#
#	under a license.  If you do not know the terms of	#
#	the license, please stop and do not read further.	#
#								#
#################################################################

diff --git a/sr_unix/gtm_dump_core.c b/sr_unix/gtm_dump_core.c
index 352581af..fa852602 100755
--- a/sr_unix/gtm_dump_core.c
+++ b/sr_unix/gtm_dump_core.c
@@ -46,7 +46,7 @@ void gtm_dump_core(void)
 	act.sa_flags = 0;
 #	endif
 	act.sa_handler = SIG_DFL;
-	sigaction(SIGQUIT, &act, 0);
+	// sigaction(SIGQUIT, &act, 0);
 
 	/* We are about to generate a core file. If one already exists on the disk,
 	   make a simplistic attempt to rename it so we can get the most useful info
